frontend/src/features/env/SwitchEnvFeedbackModal.stories.tsx

```diff
@@ -28,27 +29,11 @@ const onClose = () => {
 }
 
 const AdminTemplate: Story = () => {
-  return (
-    <SwitchEnvFeedbackModal
-      onChangeEnv={() => console.log('change env')}
-      onSubmitFeedback={async () => console.log('submit feedback')}
-      onClose={onClose}
-      isOpen={true}
-      isAdminView={true}
-    />
-  )
+  return <AdminFeedbackModal onClose={onClose} isOpen={true} />
```

Comment:
nit: props are always truthy in react don't need `true` assignment

```suggestion
  return <AdminFeedbackModal onClose={onClose} isOpen />
```